### DISCUSSION TITLE
HIGHS and PYOMO on Jupyter_Notebook

### DISCUSSION THREAD

**Discussion by OptimizationExpert** [ERGO-Code/HiGHS#1019] [Q&A] [answered]
Please advise how to use HIGHS for Pyomo on Cona Jupyter_Notebook. 
Both for Windows and Mac

**Comment by jajhall** [comment, marked as answer]
Thanks for this. We will remember to address this when we integrate "live" Python into the web-based documentation that we are now in a position to develop
